honeyscanner/Pipfile

```diff
@@ -12,6 +12,8 @@ packaging = "*"
 paramiko = "*"
 pygithub = "*"
 python3-nmap = "*"
+socket = "*"
+ssl = "*"
```

Comment:
Why do we need the `ssl` library?

Reply:
Because on port 455 in Dionaea honeypot is running http server and I want to get ssl cert which contains 'dionaea'

Reply:
The `socket = "*"` line can be removed as socket is a built-in module.